rules/manager.go

```diff
@@ -743,6 +743,37 @@ func (g *Group) RestoreForState(ts time.Time) {
 
 }
 
+// Equals return if two groups have the same
+// 1. name
+// 2. file path
+// 3. interval
+// 4. rule set. and the rules are the same
```

Comment:
Maybe simplify thisEquals return if two groups are the same.

Reply:
that's ok. the code shows the same things. please check the latest commit .